cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/customaction/PipelineAction.java

```diff
@@ -81,7 +81,9 @@ public void run() throws Exception {
                                                           context,
                                                           context.getNamespace()));
     ActionContext actionContext = new BasicActionContext(context, metrics, stageInfo.getName());
-    action.run(actionContext);
+    if (!context.getDataTracer(stageInfo.getName()).isEnabled()) {
```

Comment:
what types of plugins are not preview supported? For example, if Actions are not supported, shouldn't PostActions also not be supported?